src/components/SearchButton.vue

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+export default {
+  props: { title: { type: String }, handleClick: { type: Function } },
+  setup() {}
+}
+</script>
+
+<template>
+  <div>
+    <button class="button" @click="handleClick">{{ title }}</button>
```

Comment:
Props - it's just a data, you don't need to provide callbacks in props. Use emits to trigger event up and then catch it in the parent component.
https://vuejs.org/guide/components/events.html#component-events